roxie/ccd/ccdactivities.cpp

```diff
@@ -3244,7 +3253,7 @@ class CRoxieIndexCountActivity : public CRoxieIndexActivity
                         count += countHelper->numValid(tlk->queryKeyBuffer());
                         if (count > rowLimit)
                             limitExceeded(false);
-                        else if (count > keyedLimit)
+                        else if (keyprocessed > keyedLimit)
```

Comment:
should probably also be keyprocessed in line 3247

Reply:
Not sure where you mean - this IS 3247

Reply:
Maybe I mean 3260

Reply:
I don't think so - when there is no postfilter keyprocessed is not being updated or separately tracked.

Now you could argue that it SHOULD be (if it contributes to any statistics), but that would be a separate bug.

Reply:
I don't think it is used in stats.